core/src/main/scala/org/apache/spark/eventhubs/EventHubsConf.scala

```diff
@@ -392,26 +392,6 @@ object EventHubsConf extends Logging {
     Serialization.write[T](value)
   }
 
-  /**
```

Comment:
Are the tests still failing with this included? Is that the only motivation for removing this? 

The intention was the reduce the payload that is serialized and sent to executors (for example people with hundreds of partitions **and** they have `startingPositions` set).

Reply:
Let me check if I can come up with a test.

Reply:
I see, it gets serialized and send to the executor in here: https://github.com/Azure/azure-event-hubs-spark/blob/master/core/src/main/scala/org/apache/spark/sql/eventhubs/EventHubsWriter.scala#L73

Reply:
I've restored the trimmed configs by using `.trimmed`